client/src/components/layouts/global.css

```
@@ -131,6 +131,10 @@ a:focus {
   border-color: var(--secondary-color);
 }
 
+button {
+  background-color: var(--secondary-background);
```

```suggestion
  background-color: var(--quaternary-background);
```
we are using quaternary-background for all buttons